src/components/CardsSlider.tsx

```diff
@@ -4,7 +4,7 @@ import BankCard from './BankCard'
 import { Card } from '../typescript/types'
```

Comment:
Remove "type" key from this type. 
Also remove type from cardsList on Home page